tests/lexer/lexer_test.cpp

```diff
@@ -1,70 +1,88 @@
 #include "catch2/catch.hpp"
 #include "lexer.hpp"
+#include "token.hpp"
 #include <string>
 #include "fmt/format.h"
 #include "trie.hpp"
 #include "catch_helpers.hpp"
+#include <vector>
 
-using namespace std;
 namespace testing
 {
 
-    void checkInTrie(string key, common::Trie<int>
-                                     trie)
+    void checkInTrie(std::string key, common::Trie<TokenType>
+                                          trie)
     {
         auto result = trie.Find(key);
         CHECK_MESSAGE((bool)result,
                       "expected to find key \"{}\", but didn't", key);
     }
+
 } // namespace testing
+
+void checkTokens(std::vector<std::string> tokens, common::Trie<TokenType> trie)
+{
+    for (std::string token : tokens)
+    {
+        testing::checkInTrie(token, trie);
+    }
+}
+
+int coutInvalidTokens(std::vector<Token> vector)
+{
+    int counter = 0;
+    for (Token tk : vector)
+    {
+        if (tk.type == TokenType::InvalidToken)
+        {
+            counter++;
+        }
+    }
+    return counter;
+}
+
 SCENARIO("Trie initialization")
 {
 
     GIVEN("The file name")
     {
-        THEN("it s readen")
+        THEN("it's readen without errors")
```

Comment:
How about a short test with the particular array of expected tokens from a specific file?

Reply:
I'll add